src/components/project.js

```diff
@@ -0,0 +1,192 @@
+import { Header, HeaderLineBelow } from "../styles/Header"
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import Body from "styles/Body"
+import { SB_ORANGE, SB_NAVY_RGBA } from "@colors"
+
+const CardBG = styled.div`
+  margin-bottom: 20px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  border-radius: 20px;
+  transition: 0.3s;
+  width: 300px;
+  height: 300px;
+  text-align: center;
+  &:hover {
+    transform: translateY(-5px);
+  }
+`
+
+const Card = styled.div`
+  margin-top: -20px;
+  width: 300px;
+  height: 300px;
+  border-radius: 20px;
+  background-color: ${SB_NAVY_RGBA(0.8)};
+  cursor: pointer;
+`
+
+const CardHeader = styled(Header)`
+  padding-top: 45px;
+  font-size: 1.5em;
+  font-weight: 300;
+  color: ${SB_ORANGE};
+  height: 100px;
+  margin-left: 20px;
+  margin-right: 20px;
+`
+const TagsContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  width: 75%;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  height: 50px;
+`
+
+const Tag = styled.div`
+  font-size: 0.9em;
+  display: inline-block;
+  border-radius: 4px;
+  background-color: rgba(0, 0, 0, 0.35);
+  padding: 5px 7px 5px 7px;
+  margin: 0px 3px 6px 3px;
+  text-align: center;
+  color: white;
+`
+
+const CardButton = styled.div`
+  z-index: 999;
+  display: inline-block;
+  border: 1px solid;
+  border-radius: 5px;
+  box-shadow: 0 0 5px -1px rgba(0, 0, 0, 0.2);
+  padding: 7px 30px 7px 30px;
+  text-align: center;
+  color: white;
+  cursor: pointer;
+  transition: all 0.3s ease-in;
+  -moz-transition: all 0.3s ease-in;
```

Comment:
Gatsby might automatically add the other transition prefixes with Babel? Check in the output css.